packages/react-core/src/demos/examples/AlertGroup/AlertGroupDemo.tsx

```diff
@@ -0,0 +1,423 @@
+    if (!isActive) {
+      setOpenDropdownKey(null);
+      return;
+    }
+    setOpenDropdownKey(id);
+  };
+
+  const onDropdownSelect = () => {
+    setOpenDropdownKey(null);
+  };
+
+  const buildOverflowMessage = () => {
+    const overflow = alerts.length - maxDisplayed;
+    if (overflow > 0 && maxDisplayed > 0) {
+      return `View ${overflow} more notifications in notification drawer`;
+    }
+    return '';
+  };
+
+  const getUniqueId = () => new Date().getTime();
+
+  const getTimeCreated = () => {
+    const date = new Date();
+    return (
+      date.toDateString() +
+      ' at ' +
+      ('00' + date.getHours().toString()).slice(-2) +
+      ':' +
+      ('00' + date.getMinutes().toString()).slice(-2)
+    );
+  };
+
+  const removeAlert = (key: React.Key) => {
+    setAlerts(prevAlerts => prevAlerts.filter(alert => alert.props.id !== key.toString()));
+  };
+
+  const removeAllAlerts = () => {
+    setAlerts([]);
+  };
+
+  const addSuccessAlertNotification = () => {
+    handleNewNotification(
+      'Success alert notification',
+      'Success alert',
+      'success',
+      getUniqueId(),
+      getTimeCreated(),
+      'Success alert notification description'
+    );
+  };
+
+  const addDangerAlertNotification = () => {
+    handleNewNotification(
+      'Danger alert notification',
+      'Danger alert',
+      'danger',
+      getUniqueId(),
+      getTimeCreated(),
+      'Danger alert notification description'
+    );
+  };
+
+  const addInfoAlertNotification = () => {
+    handleNewNotification(
+      'Info alert notification',
+      'Info alert',
+      'info',
+      getUniqueId(),
+      getTimeCreated(),
+      'Info alert notification description'
+    );
+  };
+
+  const addWarningAlertNotification = () => {
+    handleNewNotification(
+      'Warning alert notification',
+      'Warning alert',
+      'warning',
+      getUniqueId(),
+      getTimeCreated(),
+      'Warning alert notification description'
+    );
+  };
+
+  const addDefaultAlertNotification = () => {
+    handleNewNotification(
+      'Default alert notification',
+      'Default alert',
+      'default',
+      getUniqueId(),
+      getTimeCreated(),
+      'Default alert notification description'
+    );
+  };
+
+  const onAlertGroupOverflowClick = () => {
+    removeAllAlerts();
+    setDrawerExpanded(true);
+  };
+
+  const onMaxDisplayedAlertsMinus = () => {
+    setMaxDisplayed(normalizeAlertsNumber(maxDisplayed - 1));
+  };
+
+  const onMaxDisplayedAlertsChange = (event: any) => {
+    setMaxDisplayed(normalizeAlertsNumber(Number(event.target.value)));
+  };
+
+  const onMaxDisplayedAlertsPlus = () => {
+    setMaxDisplayed(normalizeAlertsNumber(maxDisplayed + 1));
+  };
+
+  const normalizeAlertsNumber = (value: number) => Math.max(Math.min(value, maxAlerts), minAlerts);
+
+  const alertButtonClasses = ['pf-c-button', 'pf-m-secondary'].join(' ');
+
+  const alertButtonStyle = { marginRight: '8px' };
```

Comment:
While this styling does work when the viewport width is larger, once it shrinks down the buttons stack on top of one another. So it would be worth adding some vertical margin as well.